### ISSUE TITLE
To use the Magento setup wizard sign into your Admin account and navigate to System 

### ISSUE THREAD

**Issue by synpatic** [magento/magento2#10447]
Magento ver. 2.1.7

All tricks and commands have been tried without success in this (upgrade, static content, cache clear/flushes, compiles, reindex..) 
 
If the web setup wizard is start I will always get following  
Magento Setup - 401 Unauthorized Access
 
"Welcome to Magento Admin, your online store headquarters. 
To use the Magento setup wizard sign into your Admin account and navigate to System => Tools => Web Setup Wizard"

Also I have setup the test site from the live site backup.  There this web setup wizard  is  started, BUT there the Component manager started only once  and  now when the component manager is clicked the Wait icon is just running and asking to "Please Wait".

**Comment by korostii** [comment]
Possible duplicate of #5247

**Comment by magento-engcom-team** [comment]
@synpatic thank you for your report.
Please use the issue [reporting guidelines](https://github.com/magento/magento2/wiki/Issue-reporting-guidelines) to report an issue. Please, edit issue description and include the preconditions, the steps required to reproduce, the actual result, and the expected result.
Please, add the following information:
 - [x] Preconditions: Magento version
 - [ ] Steps to reproduce
 - [ ] Expected result
 - [ ] Actual result

**Comment by magento-engcom-team** [comment]
@synpatic we are closing this issue due to inactivity. If you'd like to update it, please reopen the issue.
